pkg/oc/cli/startbuild/startbuild.go

```diff
@@ -347,13 +347,24 @@ func (o *StartBuildOptions) Validate() error {
 	if len(o.FromBuild) != 0 && o.AsBinary {
 		// TODO: we should support this, it should be possible to clone a build to run again with new uploaded artifacts.
 		// Doing so requires introducing a new clonebinary endpoint.
-		return fmt.Errorf("Cannot use '--from-build' flag with binary builds")
+		return fmt.Errorf("cannot use '--from-build' flag with binary builds")
```

Comment:
Change in case is breaking `cmd` test.

Reply:
I've fixed the tests, errors should always be lowercase. ptal